src/net/connection/virtual_connection.rs

```diff
@@ -1,31 +1,31 @@
-use super::{ExternalAcks, LocalAckRecord};
+use net::{ExternalAcks, LocalAckRecord, NetworkQuality};
 use packet::Packet;
 use std::fmt;
 use std::net::SocketAddr;
 use std::time::{Duration, Instant};
 
 /// Contains the information about a certain 'virtual connection' over udp.
 /// This stores information about the last sequence number, dropped packages, packages waiting for acknowledgement and acknowledgements gotten from the other side.
-pub struct Connection {
+pub struct VirtualConnection {
```

Comment:
Why the rename?

Reply:
Because udp connection is not an connection it is an virtual connection not one that is really there.  gaffer on games call it also like that. So I thought if we follow his blogs then lets keep the names the same also.

Reply:
Yeah, I agree on that. It's an important distinction at this level.

Reply:
Works for me! Thanks.